app/controllers/landing_page_controller.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+# Controller for the landing page
+class LandingPageController < ApplicationController
+  def index
+    @featured_items = Settings.featured_items
```

Comment:
I'd be tempted to use the config directly in the template instead of setting an instance variable.

Reply:
Makes sense. Not any different than pulling from a locale config. I went for it.